tests/unit_tests/layout/test_telescope_position.py

```diff
@@ -55,15 +55,15 @@ def test_str(crs_wgs84, crs_local, crs_utm):
 
     tel.set_coordinates("ground", 50, -25.0, 2158.0 * u.m)
     _tcors = tel.__str__()
-    _test_string = "LST-01\t CORSIKA x(->North): 50.00 y(->West): -25.00"
+    _test_string = "LST-01\t Ground x(->North): 50.00 y(->West): -25.00"
     assert _tcors == (_test_string + "\t Alt: 2158.00")
     tel.convert_all(crs_local=crs_local, crs_wgs84=crs_wgs84)
     _tcors = tel.__str__()
     _test_string += "\t Longitude: 28.76262 Latitude: -17.89177"
     assert _tcors == (_test_string + "\t Alt: 2158.00")
     tel.convert_all(crs_local=crs_local, crs_wgs84=crs_wgs84, crs_utm=crs_utm)
     _tcors = tel.__str__()
-    _test_string = "LST-01\t CORSIKA x(->North): 50.00 y(->West): -25.00"
+    _test_string = "LST-01\t Ground x(->North): 50.00 y(->West): -25.00"
```

Comment:
I don't remember whether we discussed about replacing CORSIKA by Ground but I am afraid Ground does not pass the same information as CORSIKA. CORSIKA already comes with the definition of the coordinate system, where X points towards North and Ground is somewhat vague. If we manage to defined Ground somewhere to consider the coordinate system as CORSIKA I would agree with this chance.
<img width="748" alt="Screenshot 2024-01-23 at 13 40 55" src="https://github.com/gammasim/simtools/assets/50831535/ed57cbf1-4b45-4653-aca9-16015e552840">

Reply:
The name CORSIKA is wrong, as this is not the CORSIKA coordinate system. Ours is aligned to geographic north, while CORSIKA is aligned with magnetic north.

This was discussed in https://github.com/gammasim/simtools/issues/658 and we have changed it everywhere else in the code.

Reply:
ok!